### PR TITLE
Capstone 1

### PR DESCRIPTION
**Pull request by ibugithub**
-In this branch :

- We make the mobile version of the Global Science summit website's home page.
- We make the mobile version of the Global Science summit website's About page. 
-  We make it desktop responsive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Portfolio<a name="readme-top"></a>
 
 <div align="center">
```
Comment (ghost) [inline README.md:3]: - [ ] As required in the project, Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
  - A description of the project.
  - A demo of the project features (different pages and different screen sizes).
  - You should also highlight some interesting piece of code or something you built that you are very proud of.
 - [ ] Kindly provide the correct live demo link to your project. Currently, the link provided belongs to your portfolio

File: index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                            <a href="#"><img src="images/logo.png" alt="Logo"></a>
+                        </div>
+                            <ul class="navUl">
+                                <li class="lsNone"><a href="about.html" class="tdNone">About</a></li>
+                                <li class="lsNone"><a href="" class="tdNone">Program</a></li>
+                                <li class="lsNone"><a href="" class="tdNone">Join</a></li>
+                                <li class="lsNone"><a href="" class="tdNone">Sponsor</a></li>
+                                <li class="lsNone"><a href="" class="tdNone">News</a></li>
+                                <li class="lsNone"><a href="" class="tdNone">GSCampaign</a></li>
+                            </ul>
```
Comment (ghost) [inline index.html:37]: - [ ] Currently, on your mobile menu, I cannot navigate to the `about me section`. Kindly provide a link to the `href`  attribute that points to the about me section

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @ibugithub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good deign


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @ibugithub 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Cool structure ✔️ 
- Adherence to changes suggested ✅ 
- Nice UI design 👍

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**